data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/CracCleaner.java

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+import com.farao_community.farao.data.crac_api.*;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.Identifiable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+import static com.farao_community.farao.data.crac_util.CracCleaningFeature.CHECK_CNEC_MNEC;
+import static com.farao_community.farao.data.crac_util.CracCleaningFeature.REMOVE_UNHANDLED_CONTINGENCIES;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public class CracCleaner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CracCleaner.class);
+
+    public CracCleaner() {
+    }
+
+    public List<String> cleanCrac(Crac crac, Network network) {
```

Comment:
could you use the same pattern for CracAlilasesUtil and CracCleaner ? Here you have a classic class, while CracAliasesUtil is static, and we would therefore have in the client code : 

```
CracAliasesUtil.createAliases(crac, network);
new CracCleaner().cleanCrac(crac, network);
```

I really have no opinion on what pattern is best :man_shrugging: (help @murgeyseb :p), but it would seem cleaner to me if all utility methods of this package have the same pattern.

Reply:
@murgeyseb himself told me to do this here. So that you can do this if needed:

            CracCleaner cracCleaner = new CracCleaner();
            cracCleaner.disableFeature(CracCleaningFeature.CHECK_CNEC_MNEC);
            cracCleaner.enableFeature(CracCleaningFeature.REMOVE_UNHANDLED_CONTINGENCIES);
            cracCleaner.cleanCrac(crac, network);

Reply:
You can perhaps adapt CracAliasesUtil to get something as:
```
CracAliasesCreator cracAliasesCreator = new CracAliasesCreator();
cracAliasesCreator.createAliases(crac, network, UcteNodeMatchingRule.ALL_8_CHARACTER_EQUAL);
```
Moreover, create a method with default UcteNodeMatchingRule, because it does not seem to exist there...

Reply:
Done